app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
```

Comment:
I think this is auto generate, but is a good practice to put colors in a separate file and upcase

Reply:
Also normally we do not modify html object directly but only created class